portfolio/src/main/webapp/script.js

```diff
@@ -33,17 +33,13 @@ async function getName() {
   document.getElementById('hello-container').innerText = name;
 }
 
-async function getCommentStats() {
+async function getComment() {
   const response = await fetch('/data');
   const comments = await response.json();
-  console.log(comments);
-  //const statsListElement = document.getElementById('comment-stats-container');
-  //const commentEl = document.getElementById('comments');
   const historyEl = document.getElementById('history');
-  console.log(historyEl + "history el");
   for(var i = 0; i < comments.length; i++){
```

Comment:
For added bounds security (meaning, to avoid an off by one error) you can use

```
for (const comment of comments) {
    historyEl.appendChild(createListElement(comment));
}
```

It's totally fine if you stick with `i = 0 ....` but in that case I would switch to `for (let i` instead of `for (var i` just so that `i` doesn't get introduced at the global scope which is a silly thing in JS.